jekyll/_cci2/custom-images.md

```diff
@@ -0,0 +1,140 @@
+---
+layout: classic-docs
+title: "Building Custom Image for Docker Executor"
```

Comment:
Should be "Building a Custom Image for Docker Executor" or "Building Custom Images for Docker Executor".